tests/sentry/integrations/jira_server/test_integration.py

```diff
@@ -346,6 +345,99 @@ def test_get_create_issue_config_with_ignored_fields(self):
+        """
+        event = self.store_event(
+            data={
+                "event_id": "a" * 32,
+                "message": "message",
+                "timestamp": self.min_ago,
+                "stacktrace": copy.deepcopy(DEFAULT_EVENT_DATA["stacktrace"]),
+            },
+            project_id=self.project.id,
+        )
+        group = event.group
+
+        responses.add(
+            responses.GET,
+            f"https://jira.example.org/rest/api/2/project/{DEFAULT_PROJECT_ID}/priorityscheme",
+            json={
+                "expand": "projectKeys",
+                "self": "https://jira.example.org/rest/api/2/priorityschemes/1",
+                "id": 1,
+                "name": "Default Priority Scheme",
+                "description": "The default priority scheme that applies to all projects",
+                # Exclude the "medium" priority (id 2)
+                "optionIds": ["1", "3"],
+                "defaultScheme": False,
+            },
+        )
+
+        responses.add(
+            responses.GET,
+            "https://jira.example.org/rest/api/2/priority",
+            json=[
+                {
+                    "self": "https://jira.example.org/rest/api/2/priority/1",
+                    "statusColor": "#f15C75",
+                    "description": "Serious problem that could block progress.",
+                    "iconUrl": "https://jira-integration.getsentry.net/images/icons/priorities/high.svg",
+                    "name": "High",
+                    "id": "1",
+                },
+                {
+                    "self": "https://jira.example.org/rest/api/2/priority/2",
+                    "statusColor": "#f79232",
+                    "description": "Has the potential to affect progress.",
+                    "iconUrl": "https://jira-integration.getsentry.net/images/icons/priorities/medium.svg",
+                    "name": "Medium",
+                    "id": "2",
+                },
+                {
+                    "self": "https://jira.example.org/rest/api/2/priority/3",
+                    "statusColor": "#707070",
+                    "description": "Minor problem or easily worked around.",
+                    "iconUrl": "https://jira-integration.getsentry.net/images/icons/priorities/low.svg",
+                    "name": "Low",
+                    "id": "3",
+                },
+            ],
+        )
+
+        def get_priorities_callthrough_client(installation: JiraServerIntegration):
+            """
+            This functions returns a lambda that when called upon, returns a
+            StubJiraApiClient. The only difference is we don't mock the function
+            call to get_priorities for the stub client.
+            """
+            mock_client = StubJiraApiClient()
+            # set the mock call to the real function
+            mock_client.get_priorities = installation.get_client().get_priorities  # type: ignore
```

Comment:
mypy throws `error: Cannot assign to a method  [method-assign]` because I'm overwriting the mock_client's `get_priorities` but that's intended, as I'm replacing it with the real one. I don't know how else to fix this except ignoring it

Reply:
Ask in `discuss-python-typing`? If there's no better way please add the reason to ignore comment too.

Reply:
I think you can do `setattr(mock_client, "get_priorities", installation.get_client().get_priorities)` as a workaround and make both the tests and mypy happy. there might be a better way though so worth asking in the slack channel for sure!

Reply:
oh nice that works